### PR TITLE
feat: types and px feature flag

### PR DESCRIPTION
**Pull request by leanmendoza**
# What? 
Use the `@dcl/schemas` and add the --px temporal flag for `dcl init`

# Why? 

...

### CODE REVIEW

File: src/lib/Project.ts [hunk 1]
```diff
@@ -37,6 +38,10 @@ export class Project {
     this.workingDir = workingDir
   }
 
+  async getInfo(): Promise<ProjectInfo> {
```
Comment (gonpombo8) [inline src/lib/Project.ts:41]: do we need this getInfo fn ? Are we using it ?

**Review by gonpombo8 — COMMENTED** [review]
I think that this PR has some linter errors.
Already create a PR to run the linter at the ci, but meanwhile can we fixed them here ?

File: src/lib/Project.ts (hunk 1 above)
Comment (leanmendoza) [reply src/lib/Project.ts:41]: Not really, but I'd want to add it to use it in the `js-sdk-toolchain` and avoid redefining the same function.

**Comment by leanmendoza** [comment]
> I think that this PR has some linter errors. Already create a PR to run the linter at the ci, but meanwhile can we fixed them here ?

We can wait the other PR

File: src/lib/Project.ts (hunk 1 above)
Comment (gonpombo8) [reply src/lib/Project.ts:41]: 👍🏼 nice

**Review by gonpombo8 — APPROVED** [review]

**Pull request merged**